## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 458901`
Commit: `505587cdf642c111087967397be7424dcf2b8ab6`

---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 2 packages marked as broken and skipped:</summary>
  <ul>
    <li>picosnitch</li>
    <li>picosnitch.dist</li>
  </ul>
</details>
<details>
  <summary>:x: 11 packages failed to build:</summary>
  <ul>
    <li>alpaca</li>
    <li>python312Packages.lerobot</li>
    <li>python312Packages.lerobot.dist</li>
    <li>python312Packages.torchcrepe</li>
    <li>python312Packages.torchcrepe.dist</li>
    <li>python313Packages.ax-platform</li>
    <li>python313Packages.ax-platform.dist</li>
    <li>python313Packages.lerobot</li>
    <li>python313Packages.lerobot.dist</li>
    <li>python313Packages.torchcrepe</li>
    <li>python313Packages.torchcrepe.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 578 packages built:</summary>
  <ul>
    <li>aider-chat-full</li>
    <li>aider-chat-full.dist</li>
    <li>aider-chat-with-browser</li>
    <li>aider-chat-with-browser.dist</li>
    <li>aider-chat-with-help</li>
    <li>aider-chat-with-help.dist</li>
    <li>apkleaks</li>
    <li>apkleaks.dist</li>
    <li>beets (python313Packages.beets)</li>
    <li>beets.dist (python313Packages.beets.dist)</li>
    <li>beets.doc (python313Packages.beets.doc)</li>
    <li>beets.man (python313Packages.beets.man)</li>
    <li>calibre</li>
    <li>checkov</li>
    <li>checkov.dist</li>
    <li>contact</li>
    <li>contact.dist</li>
    <li>crewai (python313Packages.crewai)</li>
    <li>crewai.dist (python313Packages.crewai.dist)</li>
    <li>cve-bin-tool</li>
    <li>cve-bin-tool.dist</li>
    <li>das</li>
    <li>das.dist</li>
    <li>deeptools</li>
    <li>deeptools.dist</li>
    <li>exo</li>
    <li>exo.dist</li>
    <li>flattenReferencesGraph</li>
    <li>flattenReferencesGraph.dist</li>
    <li>hp2p</li>
    <li>jadx</li>
    <li>marimo (python313Packages.marimo)</li>
    <li>marimo.dist (python313Packages.marimo.dist)</li>
    <li>multiqc</li>
    <li>multiqc.dist</li>
    <li>newelle</li>
    <li>pianotrans</li>
    <li>pianotrans.dist</li>
    <li>piper-tts</li>
    <li>piper-tts.dist</li>
    <li>pitivi</li>
    <li>prowler</li>
    <li>prowler.dist</li>
    <li>python312Packages.altair</li>
    <li>python312Packages.altair.dist</li>
    <li>python312Packages.ax-platform</li>
    <li>python312Packages.ax-platform.dist</li>
    <li>python312Packages.backtesting</li>
    <li>python312Packages.backtesting.dist</li>
    <li>python312Packages.beetcamp</li>
    <li>python312Packages.beetcamp.dist</li>
    <li>python312Packages.beets</li>
    <li>python312Packages.beets-audible</li>
    <li>python312Packages.beets-audible.dist</li>
    <li>python312Packages.beets.dist</li>
    <li>python312Packages.beets.doc</li>
    <li>python312Packages.beets.man</li>
    <li>python312Packages.bokeh</li>
    <li>python312Packages.bokeh.dist</li>
    <li>python312Packages.bumps</li>
    <li>python312Packages.bumps.dist</li>
    <li>python312Packages.chart-studio</li>
    <li>python312Packages.chart-studio.dist</li>
    <li>python312Packages.coiled</li>
    <li>python312Packages.coiled.dist</li>
    <li>python312Packages.compressai</li>
    <li>python312Packages.compressai.dist</li>
    <li>python312Packages.crewai</li>
    <li>python312Packages.crewai.dist</li>
    <li>python312Packages.dalle-mini</li>
    <li>python312Packages.dalle-mini.dist</li>
    <li>python312Packages.dash</li>
    <li>python312Packages.dash-bootstrap-components</li>
    <li>python312Packages.dash-bootstrap-components.dist</li>
    <li>python312Packages.dash-bootstrap-templates</li>
    <li>python312Packages.dash-bootstrap-templates.dist</li>
    <li>python312Packages.dash.dist</li>
    <li>python312Packages.dm-sonnet</li>
    <li>python312Packages.dm-sonnet.dist</li>
    <li>python312Packages.energyflow</li>
    <li>python312Packages.energyflow.dist</li>
    <li>python312Packages.experiment-utilities</li>
    <li>python312Packages.experiment-utilities.dist</li>
    <li>python312Packages.explorerscript</li>
    <li>python312Packages.explorerscript.dist</li>
    <li>python312Packages.fastai</li>
    <li>python312Packages.fastai.dist</li>
    <li>python312Packages.formulaic</li>
    <li>python312Packages.formulaic.dist</li>
    <li>python312Packages.gradio</li>
    <li>python312Packages.gradio.dist</li>
    <li>python312Packages.great-tables</li>
    <li>python312Packages.great-tables.dist</li>
    <li>python312Packages.heudiconv</li>
    <li>python312Packages.heudiconv.dist</li>
    <li>python312Packages.holistic-trace-analysis</li>
    <li>python312Packages.holistic-trace-analysis.dist</li>
    <li>python312Packages.holoviews</li>
    <li>python312Packages.holoviews.dist</li>
    <li>python312Packages.hvplot</li>
    <li>python312Packages.hvplot.dist</li>
    <li>python312Packages.igraph</li>
    <li>python312Packages.igraph.dist</li>
    <li>python312Packages.intake</li>
    <li>python312Packages.intake.dist</li>
    <li>python312Packages.iplotx</li>
    <li>python312Packages.iplotx.dist</li>
    <li>python312Packages.itables</li>
    <li>python312Packages.itables.dist</li>
    <li>python312Packages.k-diffusion</li>
    <li>python312Packages.k-diffusion.dist</li>
    <li>python312Packages.kmapper</li>
    <li>python312Packages.kmapper.dist</li>
    <li>python312Packages.layoutparser</li>
    <li>python312Packages.layoutparser.dist</li>
    <li>python312Packages.leidenalg</li>
    <li>python312Packages.leidenalg.dist</li>
    <li>python312Packages.librosa</li>
    <li>python312Packages.librosa.dist</li>
    <li>python312Packages.lida</li>
    <li>python312Packages.lida.dist</li>
    <li>python312Packages.lifelines</li>
    <li>python312Packages.lifelines.dist</li>
    <li>python312Packages.linearmodels</li>
    <li>python312Packages.linearmodels.dist</li>
    <li>python312Packages.livelossplot</li>
    <li>python312Packages.livelossplot.dist</li>
    <li>python312Packages.llama-cloud-services</li>
    <li>python312Packages.llama-cloud-services.dist</li>
    <li>python312Packages.llama-index</li>
    <li>python312Packages.llama-index-cli</li>
    <li>python312Packages.llama-index-cli.dist</li>
    <li>python312Packages.llama-index-core</li>
    <li>python312Packages.llama-index-core.dist</li>
    <li>python312Packages.llama-index-embeddings-gemini</li>
    <li>python312Packages.llama-index-embeddings-gemini.dist</li>
    <li>python312Packages.llama-index-embeddings-google</li>
    <li>python312Packages.llama-index-embeddings-google.dist</li>
    <li>python312Packages.llama-index-embeddings-huggingface</li>
    <li>python312Packages.llama-index-embeddings-huggingface.dist</li>
    <li>python312Packages.llama-index-embeddings-ollama</li>
    <li>python312Packages.llama-index-embeddings-ollama.dist</li>
    <li>python312Packages.llama-index-embeddings-openai</li>
    <li>python312Packages.llama-index-embeddings-openai.dist</li>
    <li>python312Packages.llama-index-graph-stores-nebula</li>
    <li>python312Packages.llama-index-graph-stores-nebula.dist</li>
    <li>python312Packages.llama-index-graph-stores-neo4j</li>
    <li>python312Packages.llama-index-graph-stores-neo4j.dist</li>
    <li>python312Packages.llama-index-graph-stores-neptune</li>
    <li>python312Packages.llama-index-graph-stores-neptune.dist</li>
    <li>python312Packages.llama-index-indices-managed-llama-cloud</li>
    <li>python312Packages.llama-index-indices-managed-llama-cloud.dist</li>
    <li>python312Packages.llama-index-legacy</li>
    <li>python312Packages.llama-index-legacy.dist</li>
    <li>python312Packages.llama-index-llms-ollama</li>
    <li>python312Packages.llama-index-llms-ollama.dist</li>
    <li>python312Packages.llama-index-llms-openai</li>
    <li>python312Packages.llama-index-llms-openai-like</li>
    <li>python312Packages.llama-index-llms-openai-like.dist</li>
    <li>python312Packages.llama-index-llms-openai.dist</li>
    <li>python312Packages.llama-index-multi-modal-llms-openai</li>
    <li>python312Packages.llama-index-multi-modal-llms-openai.dist</li>
    <li>python312Packages.llama-index-node-parser-docling</li>
    <li>python312Packages.llama-index-node-parser-docling.dist</li>
    <li>python312Packages.llama-index-readers-database</li>
    <li>python312Packages.llama-index-readers-database.dist</li>
    <li>python312Packages.llama-index-readers-file</li>
    <li>python312Packages.llama-index-readers-file.dist</li>
    <li>python312Packages.llama-index-readers-json</li>
    <li>python312Packages.llama-index-readers-json.dist</li>
    <li>python312Packages.llama-index-readers-llama-parse</li>
    <li>python312Packages.llama-index-readers-llama-parse.dist</li>
    <li>python312Packages.llama-index-readers-s3</li>
    <li>python312Packages.llama-index-readers-s3.dist</li>
    <li>python312Packages.llama-index-readers-twitter</li>
    <li>python312Packages.llama-index-readers-twitter.dist</li>
    <li>python312Packages.llama-index-readers-txtai</li>
    <li>python312Packages.llama-index-readers-txtai.dist</li>
    <li>python312Packages.llama-index-readers-weather</li>
    <li>python312Packages.llama-index-readers-weather.dist</li>
    <li>python312Packages.llama-index-vector-stores-chroma</li>
    <li>python312Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python312Packages.llama-index-vector-stores-google</li>
    <li>python312Packages.llama-index-vector-stores-google.dist</li>
    <li>python312Packages.llama-index-vector-stores-milvus</li>
    <li>python312Packages.llama-index-vector-stores-milvus.dist</li>
    <li>python312Packages.llama-index-vector-stores-postgres</li>
    <li>python312Packages.llama-index-vector-stores-postgres.dist</li>
    <li>python312Packages.llama-index-vector-stores-qdrant</li>
    <li>python312Packages.llama-index-vector-stores-qdrant.dist</li>
    <li>python312Packages.llama-index.dist</li>
    <li>python312Packages.llama-parse</li>
    <li>python312Packages.llama-parse.dist</li>
    <li>python312Packages.marimo</li>
    <li>python312Packages.marimo.dist</li>
    <li>python312Packages.meshtastic</li>
    <li>python312Packages.meshtastic.dist</li>
    <li>python312Packages.mlcroissant</li>
    <li>python312Packages.mlcroissant.dist</li>
    <li>python312Packages.msticpy</li>
    <li>python312Packages.msticpy.dist</li>
    <li>python312Packages.narwhals</li>
    <li>python312Packages.narwhals.dist</li>
    <li>python312Packages.nbsmoke</li>
    <li>python312Packages.nbsmoke.dist</li>
    <li>python312Packages.neurokit2</li>
    <li>python312Packages.neurokit2.dist</li>
    <li>python312Packages.niaarm</li>
    <li>python312Packages.niaarm.dist</li>
    <li>python312Packages.nipype</li>
    <li>python312Packages.nipype.dist</li>
    <li>python312Packages.niworkflows</li>
    <li>python312Packages.niworkflows.dist</li>
    <li>python312Packages.optuna</li>
    <li>python312Packages.optuna-dashboard</li>
    <li>python312Packages.optuna-dashboard.dist</li>
    <li>python312Packages.optuna.dist</li>
    <li>python312Packages.pandantic</li>
    <li>python312Packages.pandantic.dist</li>
    <li>python312Packages.pandas-stubs</li>
    <li>python312Packages.pandas-stubs.dist</li>
    <li>python312Packages.pandera</li>
    <li>python312Packages.pandera.dist</li>
    <li>python312Packages.panel</li>
    <li>python312Packages.panel.dist</li>
    <li>python312Packages.pdfplumber</li>
    <li>python312Packages.pdfplumber.dist</li>
    <li>python312Packages.piano-transcription-inference</li>
    <li>python312Packages.piano-transcription-inference.dist</li>
    <li>python312Packages.plopp</li>
    <li>python312Packages.plopp.dist</li>
    <li>python312Packages.plotly</li>
    <li>python312Packages.plotly.dist</li>
    <li>python312Packages.pyannote-audio</li>
    <li>python312Packages.pyannote-audio.dist</li>
    <li>python312Packages.pyannote-pipeline</li>
    <li>python312Packages.pyannote-pipeline.dist</li>
    <li>python312Packages.pybids</li>
    <li>python312Packages.pybids.dist</li>
    <li>python312Packages.pymatgen</li>
    <li>python312Packages.pymatgen.dist</li>
    <li>python312Packages.pyreadstat</li>
    <li>python312Packages.pyreadstat.dist</li>
    <li>python312Packages.reflex</li>
    <li>python312Packages.reflex-chakra</li>
    <li>python312Packages.reflex-chakra.dist</li>
    <li>python312Packages.reflex.dist</li>
    <li>python312Packages.resampy</li>
    <li>python312Packages.resampy.dist</li>
    <li>python312Packages.sasmodels</li>
    <li>python312Packages.sasmodels.dist</li>
    <li>python312Packages.scikit-tda</li>
    <li>python312Packages.scikit-tda.dist</li>
    <li>python312Packages.shiny</li>
    <li>python312Packages.shiny.dist</li>
    <li>python312Packages.skops</li>
    <li>python312Packages.skops.dist</li>
    <li>python312Packages.skrl</li>
    <li>python312Packages.skrl.dist</li>
    <li>python312Packages.skytemple-dtef</li>
    <li>python312Packages.skytemple-dtef.dist</li>
    <li>python312Packages.skytemple-files</li>
    <li>python312Packages.skytemple-files.dist</li>
    <li>python312Packages.skytemple-ssb-debugger</li>
    <li>python312Packages.skytemple-ssb-debugger.dist</li>
    <li>python312Packages.smolagents</li>
    <li>python312Packages.smolagents.dist</li>
    <li>python312Packages.spacy</li>
    <li>python312Packages.spacy-curated-transformers</li>
    <li>python312Packages.spacy-curated-transformers.dist</li>
    <li>python312Packages.spacy-loggers</li>
    <li>python312Packages.spacy-loggers.dist</li>
    <li>python312Packages.spacy-lookups-data</li>
    <li>python312Packages.spacy-lookups-data.dist</li>
    <li>python312Packages.spacy-transformers</li>
    <li>python312Packages.spacy-transformers.dist</li>
    <li>python312Packages.spacy.dist</li>
    <li>python312Packages.st-pages</li>
    <li>python312Packages.st-pages.dist</li>
    <li>python312Packages.streamlit</li>
    <li>python312Packages.streamlit.dist</li>
    <li>python312Packages.sumo</li>
    <li>python312Packages.sumo.dist</li>
    <li>python312Packages.synergy</li>
    <li>python312Packages.synergy.dist</li>
    <li>python312Packages.templateflow</li>
    <li>python312Packages.templateflow.dist</li>
    <li>python312Packages.tensorflow-datasets</li>
    <li>python312Packages.tensorflow-datasets.dist</li>
    <li>python312Packages.textacy</li>
    <li>python312Packages.textacy.dist</li>
    <li>python312Packages.textnets</li>
    <li>python312Packages.textnets.dist</li>
    <li>python312Packages.tinygrad</li>
    <li>python312Packages.tinygrad.dist</li>
    <li>python312Packages.torch-audiomentations</li>
    <li>python312Packages.torch-audiomentations.dist</li>
    <li>python312Packages.torchlibrosa</li>
    <li>python312Packages.torchlibrosa.dist</li>
    <li>python312Packages.txtai</li>
    <li>python312Packages.txtai.dist</li>
    <li>python312Packages.unstructured-inference</li>
    <li>python312Packages.unstructured-inference.dist</li>
    <li>python312Packages.vega</li>
    <li>python312Packages.vega.dist</li>
    <li>python312Packages.wandb</li>
    <li>python312Packages.wandb.dist</li>
    <li>python312Packages.whisperx</li>
    <li>python312Packages.whisperx.dist</li>
    <li>python313Packages.altair</li>
    <li>python313Packages.altair.dist</li>
    <li>python313Packages.backtesting</li>
    <li>python313Packages.backtesting.dist</li>
    <li>python313Packages.beetcamp</li>
    <li>python313Packages.beetcamp.dist</li>
    <li>python313Packages.beets-audible</li>
    <li>python313Packages.beets-audible.dist</li>
    <li>python313Packages.bokeh</li>
    <li>python313Packages.bokeh.dist</li>
    <li>python313Packages.bumps</li>
    <li>python313Packages.bumps.dist</li>
    <li>python313Packages.chart-studio</li>
    <li>python313Packages.chart-studio.dist</li>
    <li>python313Packages.coiled</li>
    <li>python313Packages.coiled.dist</li>
    <li>python313Packages.compressai</li>
    <li>python313Packages.compressai.dist</li>
    <li>python313Packages.dalle-mini</li>
    <li>python313Packages.dalle-mini.dist</li>
    <li>python313Packages.dash</li>
    <li>python313Packages.dash-bootstrap-components</li>
    <li>python313Packages.dash-bootstrap-components.dist</li>
    <li>python313Packages.dash-bootstrap-templates</li>
    <li>python313Packages.dash-bootstrap-templates.dist</li>
    <li>python313Packages.dash.dist</li>
    <li>python313Packages.dm-sonnet</li>
    <li>python313Packages.dm-sonnet.dist</li>
    <li>python313Packages.energyflow</li>
    <li>python313Packages.energyflow.dist</li>
    <li>python313Packages.experiment-utilities</li>
    <li>python313Packages.experiment-utilities.dist</li>
    <li>python313Packages.explorerscript</li>
    <li>python313Packages.explorerscript.dist</li>
    <li>python313Packages.fastai</li>
    <li>python313Packages.fastai.dist</li>
    <li>python313Packages.formulaic</li>
    <li>python313Packages.formulaic.dist</li>
    <li>python313Packages.gradio</li>
    <li>python313Packages.gradio.dist</li>
    <li>python313Packages.great-tables</li>
    <li>python313Packages.great-tables.dist</li>
    <li>python313Packages.heudiconv</li>
    <li>python313Packages.heudiconv.dist</li>
    <li>python313Packages.holistic-trace-analysis</li>
    <li>python313Packages.holistic-trace-analysis.dist</li>
    <li>python313Packages.holoviews</li>
    <li>python313Packages.holoviews.dist</li>
    <li>python313Packages.hvplot</li>
    <li>python313Packages.hvplot.dist</li>
    <li>python313Packages.igraph</li>
    <li>python313Packages.igraph.dist</li>
    <li>python313Packages.intake</li>
    <li>python313Packages.intake.dist</li>
    <li>python313Packages.iplotx</li>
    <li>python313Packages.iplotx.dist</li>
    <li>python313Packages.itables</li>
    <li>python313Packages.itables.dist</li>
    <li>python313Packages.k-diffusion</li>
    <li>python313Packages.k-diffusion.dist</li>
    <li>python313Packages.kmapper</li>
    <li>python313Packages.kmapper.dist</li>
    <li>python313Packages.layoutparser</li>
    <li>python313Packages.layoutparser.dist</li>
    <li>python313Packages.leidenalg</li>
    <li>python313Packages.leidenalg.dist</li>
    <li>python313Packages.librosa</li>
    <li>python313Packages.librosa.dist</li>
    <li>python313Packages.lida</li>
    <li>python313Packages.lida.dist</li>
    <li>python313Packages.lifelines</li>
    <li>python313Packages.lifelines.dist</li>
    <li>python313Packages.linearmodels</li>
    <li>python313Packages.linearmodels.dist</li>
    <li>python313Packages.livelossplot</li>
    <li>python313Packages.livelossplot.dist</li>
    <li>python313Packages.llama-cloud-services</li>
    <li>python313Packages.llama-cloud-services.dist</li>
    <li>python313Packages.llama-index</li>
    <li>python313Packages.llama-index-cli</li>
    <li>python313Packages.llama-index-cli.dist</li>
    <li>python313Packages.llama-index-core</li>
    <li>python313Packages.llama-index-core.dist</li>
    <li>python313Packages.llama-index-embeddings-gemini</li>
    <li>python313Packages.llama-index-embeddings-gemini.dist</li>
    <li>python313Packages.llama-index-embeddings-google</li>
    <li>python313Packages.llama-index-embeddings-google.dist</li>
    <li>python313Packages.llama-index-embeddings-huggingface</li>
    <li>python313Packages.llama-index-embeddings-huggingface.dist</li>
    <li>python313Packages.llama-index-embeddings-ollama</li>
    <li>python313Packages.llama-index-embeddings-ollama.dist</li>
    <li>python313Packages.llama-index-embeddings-openai</li>
    <li>python313Packages.llama-index-embeddings-openai.dist</li>
    <li>python313Packages.llama-index-graph-stores-neo4j</li>
    <li>python313Packages.llama-index-graph-stores-neo4j.dist</li>
    <li>python313Packages.llama-index-graph-stores-neptune</li>
    <li>python313Packages.llama-index-graph-stores-neptune.dist</li>
    <li>python313Packages.llama-index-indices-managed-llama-cloud</li>
    <li>python313Packages.llama-index-indices-managed-llama-cloud.dist</li>
    <li>python313Packages.llama-index-legacy</li>
    <li>python313Packages.llama-index-legacy.dist</li>
    <li>python313Packages.llama-index-llms-ollama</li>
    <li>python313Packages.llama-index-llms-ollama.dist</li>
    <li>python313Packages.llama-index-llms-openai</li>
    <li>python313Packages.llama-index-llms-openai-like</li>
    <li>python313Packages.llama-index-llms-openai-like.dist</li>
    <li>python313Packages.llama-index-llms-openai.dist</li>
    <li>python313Packages.llama-index-multi-modal-llms-openai</li>
    <li>python313Packages.llama-index-multi-modal-llms-openai.dist</li>
    <li>python313Packages.llama-index-node-parser-docling</li>
    <li>python313Packages.llama-index-node-parser-docling.dist</li>
    <li>python313Packages.llama-index-readers-database</li>
    <li>python313Packages.llama-index-readers-database.dist</li>
    <li>python313Packages.llama-index-readers-file</li>
    <li>python313Packages.llama-index-readers-file.dist</li>
    <li>python313Packages.llama-index-readers-json</li>
    <li>python313Packages.llama-index-readers-json.dist</li>
    <li>python313Packages.llama-index-readers-llama-parse</li>
    <li>python313Packages.llama-index-readers-llama-parse.dist</li>
    <li>python313Packages.llama-index-readers-s3</li>
    <li>python313Packages.llama-index-readers-s3.dist</li>
    <li>python313Packages.llama-index-readers-twitter</li>
    <li>python313Packages.llama-index-readers-twitter.dist</li>
    <li>python313Packages.llama-index-readers-txtai</li>
    <li>python313Packages.llama-index-readers-txtai.dist</li>
    <li>python313Packages.llama-index-readers-weather</li>
    <li>python313Packages.llama-index-readers-weather.dist</li>
    <li>python313Packages.llama-index-vector-stores-chroma</li>
    <li>python313Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python313Packages.llama-index-vector-stores-google</li>
    <li>python313Packages.llama-index-vector-stores-google.dist</li>
    <li>python313Packages.llama-index-vector-stores-milvus</li>
    <li>python313Packages.llama-index-vector-stores-milvus.dist</li>
    <li>python313Packages.llama-index-vector-stores-postgres</li>
    <li>python313Packages.llama-index-vector-stores-postgres.dist</li>
    <li>python313Packages.llama-index-vector-stores-qdrant</li>
    <li>python313Packages.llama-index-vector-stores-qdrant.dist</li>
    <li>python313Packages.llama-index.dist</li>
    <li>python313Packages.llama-parse</li>
    <li>python313Packages.llama-parse.dist</li>
    <li>python313Packages.meshtastic</li>
    <li>python313Packages.meshtastic.dist</li>
    <li>python313Packages.mlcroissant</li>
    <li>python313Packages.mlcroissant.dist</li>
    <li>python313Packages.msticpy</li>
    <li>python313Packages.msticpy.dist</li>
    <li>python313Packages.narwhals</li>
    <li>python313Packages.narwhals.dist</li>
    <li>python313Packages.nbsmoke</li>
    <li>python313Packages.nbsmoke.dist</li>
    <li>python313Packages.neurokit2</li>
    <li>python313Packages.neurokit2.dist</li>
    <li>python313Packages.niaarm</li>
    <li>python313Packages.niaarm.dist</li>
    <li>python313Packages.nipype</li>
    <li>python313Packages.nipype.dist</li>
    <li>python313Packages.niworkflows</li>
    <li>python313Packages.niworkflows.dist</li>
    <li>python313Packages.optuna</li>
    <li>python313Packages.optuna-dashboard</li>
    <li>python313Packages.optuna-dashboard.dist</li>
    <li>python313Packages.optuna.dist</li>
    <li>python313Packages.pandantic</li>
    <li>python313Packages.pandantic.dist</li>
    <li>python313Packages.pandas-stubs</li>
    <li>python313Packages.pandas-stubs.dist</li>
    <li>python313Packages.pandera</li>
    <li>python313Packages.pandera.dist</li>
    <li>python313Packages.panel</li>
    <li>python313Packages.panel.dist</li>
    <li>python313Packages.pdfplumber</li>
    <li>python313Packages.pdfplumber.dist</li>
    <li>python313Packages.piano-transcription-inference</li>
    <li>python313Packages.piano-transcription-inference.dist</li>
    <li>python313Packages.plopp</li>
    <li>python313Packages.plopp.dist</li>
    <li>python313Packages.plotly</li>
    <li>python313Packages.plotly.dist</li>
    <li>python313Packages.pyannote-audio</li>
    <li>python313Packages.pyannote-audio.dist</li>
    <li>python313Packages.pyannote-pipeline</li>
    <li>python313Packages.pyannote-pipeline.dist</li>
    <li>python313Packages.pybids</li>
    <li>python313Packages.pybids.dist</li>
    <li>python313Packages.pyreadstat</li>
    <li>python313Packages.pyreadstat.dist</li>
    <li>python313Packages.reflex</li>
    <li>python313Packages.reflex-chakra</li>
    <li>python313Packages.reflex-chakra.dist</li>
    <li>python313Packages.reflex.dist</li>
    <li>python313Packages.resampy</li>
    <li>python313Packages.resampy.dist</li>
    <li>python313Packages.sasmodels</li>
    <li>python313Packages.sasmodels.dist</li>
    <li>python313Packages.scikit-tda</li>
    <li>python313Packages.scikit-tda.dist</li>
    <li>python313Packages.shiny</li>
    <li>python313Packages.shiny.dist</li>
    <li>python313Packages.skops</li>
    <li>python313Packages.skops.dist</li>
    <li>python313Packages.skrl</li>
    <li>python313Packages.skrl.dist</li>
    <li>python313Packages.skytemple-dtef</li>
    <li>python313Packages.skytemple-dtef.dist</li>
    <li>python313Packages.skytemple-files</li>
    <li>python313Packages.skytemple-files.dist</li>
    <li>python313Packages.skytemple-ssb-debugger</li>
    <li>python313Packages.skytemple-ssb-debugger.dist</li>
    <li>python313Packages.smolagents</li>
    <li>python313Packages.smolagents.dist</li>
    <li>python313Packages.spacy</li>
    <li>python313Packages.spacy-curated-transformers</li>
    <li>python313Packages.spacy-curated-transformers.dist</li>
    <li>python313Packages.spacy-loggers</li>
    <li>python313Packages.spacy-loggers.dist</li>
    <li>python313Packages.spacy-lookups-data</li>
    <li>python313Packages.spacy-lookups-data.dist</li>
    <li>python313Packages.spacy-transformers</li>
    <li>python313Packages.spacy-transformers.dist</li>
    <li>python313Packages.spacy.dist</li>
    <li>python313Packages.st-pages</li>
    <li>python313Packages.st-pages.dist</li>
    <li>streamlit (python313Packages.streamlit)</li>
    <li>streamlit.dist (python313Packages.streamlit.dist)</li>
    <li>python313Packages.synergy</li>
    <li>python313Packages.synergy.dist</li>
    <li>python313Packages.templateflow</li>
    <li>python313Packages.templateflow.dist</li>
    <li>python313Packages.tensorflow-datasets</li>
    <li>python313Packages.tensorflow-datasets.dist</li>
    <li>python313Packages.textacy</li>
    <li>python313Packages.textacy.dist</li>
    <li>python313Packages.textnets</li>
    <li>python313Packages.textnets.dist</li>
    <li>python313Packages.tinygrad</li>
    <li>python313Packages.tinygrad.dist</li>
    <li>python313Packages.torch-audiomentations</li>
    <li>python313Packages.torch-audiomentations.dist</li>
    <li>python313Packages.torchlibrosa</li>
    <li>python313Packages.torchlibrosa.dist</li>
    <li>python313Packages.txtai</li>
    <li>python313Packages.txtai.dist</li>
    <li>python313Packages.unstructured-inference</li>
    <li>python313Packages.unstructured-inference.dist</li>
    <li>python313Packages.vega</li>
    <li>python313Packages.vega.dist</li>
    <li>python313Packages.wandb</li>
    <li>python313Packages.wandb.dist</li>
    <li>whisperx (python313Packages.whisperx)</li>
    <li>whisperx.dist (python313Packages.whisperx.dist)</li>
    <li>quark-engine</li>
    <li>quark-engine.dist</li>
    <li>rembg</li>
    <li>rembg.dist</li>
    <li>skytemple</li>
    <li>skytemple.dist</li>
    <li>supergee</li>
    <li>theharvester</li>
    <li>theharvester.dist</li>
    <li>tocpdf</li>
    <li>tocpdf.dist</li>
    <li>tts</li>
    <li>tts.dist</li>
    <li>unbook</li>
    <li>unstructured-api</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
    <li>wyoming-piper</li>
    <li>wyoming-piper.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>python312Packages.lerobot</summary>
<pre>tests/datasets/test_dataset_utils.py ........                            [ 12%]
tests/datasets/test_datasets.py ........................ss...........    [ 16%]
tests/datasets/test_delta_timestamps.py ......                           [ 16%]
tests/datasets/test_image_transforms.py .............................    [ 19%]
tests/datasets/test_image_writer.py ................................     [ 22%]
tests/datasets/test_online_buffer.py .........                           [ 22%]
tests/datasets/test_quantiles_dataset_integration.py ......              [ 23%]
tests/datasets/test_sampler.py ....                                      [ 23%]
tests/datasets/test_streaming.py .............                           [ 24%]
tests/datasets/test_visualize_dataset.py s                               [ 25%]
tests/envs/test_envs.py sssssssss                                        [ 25%]
tests/motors/test_motors_bus.py .............................            [ 28%]
tests/optim/test_optimizers.py ............                              [ 29%]
tests/optim/test_schedulers.py ..                                        [ 29%]
tests/policies/hilserl/test_modeling_classifier.py .ssss                 [ 30%]
tests/policies/pi0_pi05/test_pi0.py ss                                   [ 30%]
tests/policies/pi0_pi05/test_pi05.py ss                                  [ 30%]
tests/policies/test_policies.py ....ssssssss...                          [ 31%]
tests/policies/test_sac_config.py ..........                             [ 32%]
tests/policies/test_sac_policy.py ......../nix/store/zmxd2m3pzig3zz9lasc61qf36nndhmgm-pytest-check-hook/nix-support/setup-hook: line 23:  1378 Killed                     /nix/store/qlz1jypzdb320jp0ci0igxg2ghf0974d-python3-3.12.11/bin/python3.12 &quot;${flagsArray[@]}&quot;</pre>
</details>
<details>
<summary>python312Packages.torchcrepe</summary>
<pre>        try:
            from torchcodec.decoders import AudioDecoder
        except ImportError as e:
&gt;           raise ImportError(
                &quot;TorchCodec is required for load_with_torchcodec. &quot; &quot;Please install torchcodec to use this function.&quot;
            ) from e
E           ImportError: TorchCodec is required for load_with_torchcodec. Please install torchcodec to use this function.

/nix/store/5rzyfgbgyv3lq26dr4fq7zvpn968zi7i-python3.12-torchaudio-2.9.0/lib/python3.12/site-packages/torchaudio/_torchcodec.py:84: ImportError
=============================== warnings summary ===============================
tests/test_core.py::test_infer_tiny
tests/test_core.py::test_infer_tiny
  /build/source/tests/test_core.py:25: DeprecationWarning: __array_wrap__ must accept context and return_scalar arguments (positionally) in the future. (Deprecated NumPy 2.0)
    diff = np.abs(activation - activation_tiny)

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
ERROR tests/test_core.py::test_embed_tiny - ImportError: TorchCodec is required for load_with_torchcodec. Please install torchcodec to use this function.
ERROR tests/test_core.py::test_embed_full - ImportError: TorchCodec is required for load_with_torchcodec. Please install torchcodec to use this function.
============= 5 passed, 1 skipped, 2 warnings, 2 errors in 15.44s ==============</pre>
</details>
<details>
<summary>python313Packages.ax-platform</summary>
<pre>  invalid value encountered in divide

ax/service/tests/test_report_utils.py::ReportUtilsTest::test_skip_contour_high_dimensional
  /build/source/ax/service/tests/test_report_utils.py:512: DeprecationWarning:
  
  Passing in a `search_space` to initialize a generator from a registry is being deprecated. `experiment` is now a required input for initializing `Adapters`. Please use `experiment` when initializing generators going forward. Support for `search_space` will be removed in Ax 0.7.0.

ax/utils/testing/tests/test_mock.py::TestMock::test_optimize_with_nsgaii_mocks
  /nix/store/x1rdpx37v2q51yvcyjk531fajgd5drhh-python3.13-botorch-0.16.0/lib/python3.13/site-packages/botorch/acquisition/analytic.py:340: NumericsWarning:
  
  ExpectedImprovement has known numerical issues that lead to suboptimal optimization performance. It is strongly recommended to simply replace
  
  	 ExpectedImprovement 	 --&gt; 	 LogExpectedImprovement 
  
  instead, which fixes the issues and has the same API. See https://arxiv.org/abs/2310.20708 for details.

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED ax/adapter/tests/test_torch_adapter.py::TorchAdapterTest::test_fitting_auxiliary_experiment_dataset - TimeoutError: Test took longer than 120 seconds. To see a profiler output, set `TestCase.PROFILE_TESTS` to `True`. To specify a reason for a long running test, utilize the @ax_long_test decorator. If your test is long because it&#x27;s doing modeling, please use the @mock_botorch_optimize decorator and see if that helps.
== 1 failed, 1473 passed, 100 deselected, 7180 warnings in 1187.43s (0:19:47) ==</pre>
</details>
<details>
<summary>python313Packages.lerobot</summary>
<pre>tests/datasets/test_dataset_utils.py ........                            [ 12%]
tests/datasets/test_datasets.py ........................ss...........    [ 16%]
tests/datasets/test_delta_timestamps.py ......                           [ 16%]
tests/datasets/test_image_transforms.py .............................    [ 19%]
tests/datasets/test_image_writer.py ................................     [ 22%]
tests/datasets/test_online_buffer.py .........                           [ 22%]
tests/datasets/test_quantiles_dataset_integration.py ......              [ 23%]
tests/datasets/test_sampler.py ....                                      [ 23%]
tests/datasets/test_streaming.py .............                           [ 24%]
tests/datasets/test_visualize_dataset.py s                               [ 25%]
tests/envs/test_envs.py sssssssss                                        [ 25%]
tests/motors/test_motors_bus.py .............................            [ 28%]
tests/optim/test_optimizers.py ............                              [ 29%]
tests/optim/test_schedulers.py ..                                        [ 29%]
tests/policies/hilserl/test_modeling_classifier.py .ssss                 [ 30%]
tests/policies/pi0_pi05/test_pi0.py ss                                   [ 30%]
tests/policies/pi0_pi05/test_pi05.py ss                                  [ 30%]
tests/policies/test_policies.py ....ssssssss...                          [ 31%]
tests/policies/test_sac_config.py ..........                             [ 32%]
tests/policies/test_sac_policy.py ......../nix/store/44nfmrh2ny1csdk0z2xz3lj1s83l6aj6-pytest-check-hook/nix-support/setup-hook: line 23:  1379 Killed                     /nix/store/cfapjd2rvqrpry4grb0kljnp8bvnvfxz-python3-3.13.8/bin/python3.13 &quot;${flagsArray[@]}&quot;</pre>
</details>
<details>
<summary>python313Packages.torchcrepe</summary>
<pre>        try:
            from torchcodec.decoders import AudioDecoder
        except ImportError as e:
&gt;           raise ImportError(
                &quot;TorchCodec is required for load_with_torchcodec. &quot; &quot;Please install torchcodec to use this function.&quot;
            ) from e
E           ImportError: TorchCodec is required for load_with_torchcodec. Please install torchcodec to use this function.

/nix/store/85xy60bqj6p0crhvm7kadynk15fyyrrq-python3.13-torchaudio-2.9.0/lib/python3.13/site-packages/torchaudio/_torchcodec.py:84: ImportError
=============================== warnings summary ===============================
tests/test_core.py::test_infer_tiny
tests/test_core.py::test_infer_tiny
  /build/source/tests/test_core.py:25: DeprecationWarning: __array_wrap__ must accept context and return_scalar arguments (positionally) in the future. (Deprecated NumPy 2.0)
    diff = np.abs(activation - activation_tiny)

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
ERROR tests/test_core.py::test_embed_tiny - ImportError: TorchCodec is required for load_with_torchcodec. Please install torchcodec to use this function.
ERROR tests/test_core.py::test_embed_full - ImportError: TorchCodec is required for load_with_torchcodec. Please install torchcodec to use this function.
============= 5 passed, 1 skipped, 2 warnings, 2 errors in 14.96s ==============</pre>
</details>
</details>

--- a/pkgs/development/python-modules/narwhals/default.nix
+++ b/pkgs/development/python-modules/narwhals/default.nix
@@ -1,5 +1,6 @@
 {
   lib,
+  stdenv,
   buildPythonPackage,
   dask,
   duckdb,
@@ -80,6 +81,11 @@ buildPythonPackage rec {
     # ibis improvements cause strict XPASS failures (tests expected to fail now pass)
     "test_empty_scalar_reduction_with_columns"
     "test_collect_empty"
+  ];
+
+  disabledTestPaths = lib.optionals (stdenv.hostPlatform.isDarwin && stdenv.hostPlatform.isx86_64) [
+    # Segfault in included polars/lazyframe
+    "tests/tpch_q1_test.py"
   ];
 
   pytestFlags = [

